variables.tf

```diff
@@ -813,3 +813,9 @@ variable "asg_terminate_lifecycle_lambda_timeout" {
   default     = 30
   type        = number
 }
+
+variable "runner_yum_update" {
```

Comment:
Can you convert the variable to a boolean instead of the string.

Reply:
Variable is now boolean.